trillian/ctfe/cert_checker_test.go

```diff
@@ -153,6 +153,130 @@ func TestValidateChain(t *testing.T) {
 	}
 }
 
+func TestCA(t *testing.T) {
+	fakeCARoots := NewPEMCertPool()
+	if !fakeCARoots.AppendCertsFromPEM([]byte(testonly.FakeCACertPEM)) {
+		t.Fatal("failed to load fake root")
+	}
+	validateOpts := CertValidationOpts{
+		trustedRoots: fakeCARoots,
+		extKeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageAny},
+	}
+	chain := pemsToDERChain(t, []string{testonly.LeafSignedByFakeIntermediateCertPEM, testonly.FakeIntermediateCertPEM})
+	leaf, err := x509.ParseCertificate(chain[0])
+	if err != nil {
+		t.Fatalf("Failed to parse golden certificate DER: %v", err)
+	}
+	t.Logf("Cert expiry date: %v", leaf.NotAfter)
```

Comment:
kinda - I was printing it just to I knew what it was when tests failed :)
Though that info is also part of the error msg, so I ding'd it.